docs/user/exports.rst

```diff
@@ -0,0 +1,90 @@
+--------
+Artemis offers several options to export or archive different data. The following table gives an overview of the available export options.
+
+.. list-table:: Export options
+   :widths: 100
+   :header-rows: 1
+
+   * - Export/Archive option
+   * - Archive course/exam
+   * - Export programming exercise material
+   * - Export programming exercise student repositories
+   * - Export exercise results
+   * - Export quiz questions
+   * - Export exercise submissions
+   * - Export user data
+
+Archive course/exam
+-------------------
+Export all course/exam data, including all exercises and student submissions.
+To archive a course or an exam the end date of the entity needs to be in the past.
+You can archive a course by clicking |archive_course| on the course management overview page or an exam by clicking |archive_exam| on the exam checklist page. This will create a zip file containing all relevant data.
+The creation is done asynchronously. You will receive a notification once the archive is ready to download. You can then download the archive by clicking |download_archive| on the course management overview page or the exam checklist page.
+
+Export programming exercise material
+------------------------------------
+Export the exercise material (template,solution,test, auxiliary repositories and problem statement and other relevant information such as points) of a programming exercise.
+To export the material click the |download_exercise| button on the exercise details page.
+
+
+Export programming exercise student repositories
+------------------------------------------------
+Export the student repositories (this can include the repositories for both graded and practice participations) of a programming exercise
+To export the repositories click the |export| button and then the |download_repos| button on the |download_scores| page.
+
+Export exercise results
+-----------------------------------
+Export the results of a specific exercise. This is supported for all exercise types.
+To export the results click the |export| button and then the |export_results| button on the |download_scores| page.
+
+Export quiz exercise
+--------------------
+Exports the questions and the sample solution of a quiz in JSON format
+You can export a a quiz exercise by clicking the |export_quiz| button on the exercises overview page.
+
+Export exercise submissions
+---------------------------
+Export the submissions of all students that participated in a specific exercise. This is supported for text, modeling and file upload exercises.
+Text submissions are exported as a zip file containing all submissions as text files.
+Modeling submissions are exported as a zip file containing all submissions as json files.
+File upload submissions are exported as a zip file containing all submitted files.
+To export the submissions click the |export_scores| button and then the |export_submissions| button on the |download_scores| page.
+
+Export course/exam scores
```

Comment:
nitpick: reordering to be after ‘Export exercise results‘ would make sense to me since it is similar functionality.